internal/tools/networkagent/main.go

```diff
@@ -250,7 +250,7 @@ func (s *service) teardownConfigureContainerNetworking(ctx context.Context, req
 
 	for _, endpoint := range resp.Endpoints {
 		if endpoint == nil {
-			log.G(ctx).WithField("name", endpoint.ID).Warn("failed to find endpoint to delete")
+			log.G(ctx).Warn("failed to find endpoint to delete")
```

Comment:
Just curious, is it possible to receive `nil` endpoints in a valid response from ncproxy?

Reply:
I'm not too sure, but I think absent a guarantee of such, the nil-check and logging is probably worth keeping.
@katiewasnothere, thought?

Reply:
I don't think it's possible but also this is just a test tool so I don't think we need to worry too much either way.